cmake/CabanaConfig.cmakein

```diff
@@ -20,4 +23,5 @@ endif()
 if(@ARBORX_FOUND@)
   find_dependency(ArborX REQUIRED)
```

Comment:
`CABANA_ENABLE_ARBORX`, matching MPI, would be helpful

Reply:
All caps `CABANA`?